help/develop/javascript-development/debugging.md

```diff
@@ -0,0 +1,103 @@
+# Debugging
+
+This document discusses debugging of your packages for Datagrok in two popular IDEs &mdash; Visual Studio Code and 
+JetBrains WebStorm.
+
+## One-click debugging with Visual Studio Code
+
+To configure VS Code for Datagrok development, run the `grok create` command with the `--ide=vscode` flag:
+
+```shell
+grok create my-datagrok-package --ide=vscode
+```
+
+**IMPORTANT**: The `--ide` flag only works on Windows. If you're using Linux or macOS, refer to the
+[Creating a VS Code Configuration for Linux or macOS section].
+
+The Grok CLI will create an additional `.vscode` folder with some useful configurations. Here are two more steps to
```

Comment:
Same remark for 'Grok CLI', let's change/not change this naming according to the above discussion.
Also, let's specify which configurations will be added in particular (`launch.json` and `tasks.json`) and give a link to VS Code documentation on debugging in case the developer wants to make additional adjustments (e.g. https://code.visualstudio.com/docs/editor/debugging).